### PR TITLE
[docs] Issue 19235

### PR DESCRIPTION
**Pull request by ddhodge**
#19235
#17989 
#17584 
DOC-147

@netlify /stable/reference/configuration/yb-tserver/

### CODE REVIEW

File: docs/content/v2.14/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Debezium supports databases with UTF-8 character encoding only. With a single-by
 
 ## Release series 1.9.5.y
```
Comment (aishwarya24) [inline docs/content/v2.14/explore/change-data-capture/debezium-connector-yugabytedb.md:45]: Why is this section not in preview and stable ? Can you add it if its supported in 2.19 and 2.18?

**Review by aishwarya24 — COMMENTED** [review]
Added @dataindataout to review #19235.

File: docs/content/v2.14/explore/change-data-capture/debezium-connector-yugabytedb.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/v2.14/explore/change-data-capture/debezium-connector-yugabytedb.md:45]: Section completely revamped for 2.18. Address in another ticket.

**Review by jasonriddell — APPROVED** [review]
From my perspective the changes look good to go.

**Pull request merged**